packages/labs/analyzer/src/lib/model.ts

```diff
@@ -413,6 +418,16 @@ export class ClassField extends Declaration {
   }
 }
 
+export class CustomElementField extends ClassField {
+  attribute?: string | boolean | undefined;
```

Comment:
afaict this is always a string in CEM. presumably you added a boolean to say "this is an attribute, by its field name"?

its probably fine but would be good to understand. i do wonder if we should purposely just duplicate the name if that's the case too, i.e. `{attribute: 'foo', name: 'foo'}`

Reply:
Yes, I found I had to pass the boolean through so that gen-manifest and friends could correctly calculate the actual attribute down the libe

Reply:
my point is maybe we should just always pass the attribute when there is one

right now:

- `name` and `attribute: false` (or `undefined`) means the field has no attribute
- `name` and `attribute: true` means the field has an attribute by the same name
- `name` and `attribute: str` means the field has an attribute named `str`

maybe we should just set it right to avoid later computation etc, as in `{attribute?: string}`.

```ts
{
  name: "my-field",
  attribute: "my-field"
}
```

etc...

Reply:
ok that sounds good. I made that change. the analyzer's `CustomElementField` instance will always get the computable string attribute when it's constructed